airbyte-integrations/connectors/destination-milvus/destination_milvus/destination.py

```diff
@@ -18,15 +25,24 @@
 BATCH_SIZE = 128
 
 
-embedder_map = {"openai": OpenAIEmbedder, "cohere": CohereEmbedder, "fake": FakeEmbedder, "from_field": FromFieldEmbedder}
+embedder_map = {
+    "openai": OpenAIEmbedder,
+    "cohere": CohereEmbedder,
+    "fake": FakeEmbedder,
+    "azure_openai": AzureOpenAIEmbedder,
+    "from_field": FromFieldEmbedder,
+}
 
 
 class DestinationMilvus(Destination):
     indexer: Indexer
     embedder: Embedder
 
     def _init_indexer(self, config: ConfigModel):
-        self.embedder = embedder_map[config.embedding.mode](config.embedding)
+        if config.embedding.mode == "azure_openai" or config.embedding.mode == "openai":
+            self.embedder = embedder_map[config.embedding.mode](config.embedding, config.processing.chunk_size)
+        else:
+            self.embedder = embedder_map[config.embedding.mode](config.embedding)
         self.indexer = MilvusIndexer(config.indexing, self.embedder.embedding_dimensions)
```

Comment:
Should this and `embedder_map` be moved into the CDK?

Not a blocker for this round, but it seems it would be good to have this logic centrally managed. (Also appears for Milvus.)

Reply:
Yeah, I think that would make sense. I'll create an issue for that